build/run-tests.sh

```diff
@@ -3,4 +3,4 @@
 set -ex
 
 # shellcheck disable=SC1010
-lein with-profile -dev do clean, javac, split-test :no-gen
+lein with-profile -dev,+ci do clean, javac, split-test :no-gen
```

Comment:
Had forgotten this in my original PR, added now.

Perhaps this file could be discarded in favor of a Lein alias? That way we have a single point of change - project.clj

Reply:
I'd be in favor of moving the logic to project.clj but keeping the file.